waveform-django/waveforms/dash_apps/finished_apps/waveform_vis.py

```diff
@@ -745,10 +762,26 @@ def get_record_event_options(click_submit, click_previous, click_next,
     # Determine what triggered this function
     ctx = dash.callback_context
     # Prepare to return the record and event value
+
+    csv_path = os.path.join(PROJECT_PATH, base.RECORDS_CSV)
```

Comment:
This chunk of code is repeated on line 452 so should probably be moved to an outside function